API/Controllers/PortfolioController.cs

```diff
@@ -0,0 +1,498 @@
+using API.Common;
+using API.Extensions;
+using API.HelperClasses;
+using API.Resources;
+using AutoMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Models;
+using Models.Defaults;
+using Serilog;
+using Services.Services;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+
+namespace API.Controllers
+{
+    /// <summary>
+    /// This class is responsible for handling HTTP requests that are related
+    /// to the portfolio and portfolio items, for example creating, retrieving, updating or deleting.
+    /// </summary>
+    [Route("api/[controller]")]
+    [ApiController]
+    public class PortfolioController : ControllerBase
+    {
+        private readonly IMapper mapper;
+        private readonly IPortfolioService portfolioService;
+        private readonly IPortfolioItemService portfolioItemService;
+        private readonly IProjectService projectService;
+        private readonly IUserService userService;
+        private readonly IAuthorizationHelper authorizationHelper;
```

Comment:
The authorization helper is not used. Although, it could give added value. Consider if the data officer should be able to make requests in name of a user in their institute (in my eyes this could absolutely work)